news.xml

```diff
@@ -556,10 +556,10 @@ $ nix-store -qR /run/current-system | grep openssl
     </title>
     <description>
       <a href="https://www.domenkozar.com/">Domen Kožar</a> gave <a
-        href="https://fosdem.org/2014/schedule/event/nixos_declarative_configuration_linux_distribution/">a
+        href="https://web.archive.org/web/20200424105824/https://archive.fosdem.org/2014/schedule/event/nixos_declarative_configuration_linux_distribution/">a
         talk at FOSDEM about NixOS</a> (<a
         href="https://video.fosdem.org/2014/H1309_Van_Rijn/Saturday/NixOS_declarative_configuration_Linux_distribution.webm">video</a>).
-      Also, Ludovic Courtès gave <a href="https://fosdem.org/2014/schedule/event/gnuguix/">a talk on
```

Comment:
https://archive.fosdem.org/2014/schedule/event/gnuguix/

Reply:
static site that won’t change